src/scripts/hero-animation/TweenLite.min.js

```diff
@@ -0,0 +1,12 @@
+/*!
```

Comment:
This file appears to come from the "GreenSock Animation Platform" - and a rather old version of it. Rather than taking a snapshot of the JS file, it might be better to instead add it as an NPM dependency: https://www.npmjs.com/package/gsap

Probably needs to be part of the wider JS team discussion though, since we'd need to decide whether we import from there or copy the JS files at build time.

Reply:
Same thing goes for `TweenLite.min.js`, btw.

Reply:
@c1rrus I've made the non-JS changes now. Sure let's discuss those with the wider JS team:)